src/beman/execution26/CMakeLists.txt

```diff
@@ -40,7 +40,10 @@ target_sources(
          ${PROJECT_SOURCE_DIR}/include/beman/execution26/detail/as_awaitable.hpp
          ${PROJECT_SOURCE_DIR}/include/beman/execution26/detail/as_except_ptr.hpp
          ${PROJECT_SOURCE_DIR}/include/beman/execution26/detail/await_result_type.hpp
+         ${PROJECT_SOURCE_DIR}/include/beman/execution26/detail/atomic_intrusive_stack.hpp
+         ${PROJECT_SOURCE_DIR}/include/beman/execution26/detail/await_result_type.hpp
```

Comment:
It is not clear to me why this additional  headers are needed now?

The last CI runs worked without this headers???

We have to check the CI build rules!

Reply:
We always need to install headers since this is a header only library. We simply forgot to add them to the file set when I implemented the awaitable helpers.

The things that I change here occured while I was using `beman::execution26` as an installed library to implement a coroutine task. I couldn't use the library because headers were missing.

I was using it here
https://github.com/maikel/task

Reply:
This is how I expect to consume beman.execution26

https://github.com/maikel/task/blob/main/CMakeLists.txt#L20-L26